uninstall.php

```diff
@@ -27,20 +25,10 @@
  * wp-config.php. This is to prevent data loss when deleting the plugin from the backend
  * and to ensure only the site owner can perform this action.
  */
-if ( Constants::is_true( 'WC_REMOVE_ALL_DATA' ) ) {
-	/**
-	 * Load core packages autoloader.
-	 */
-	if ( ! class_exists( 'Automattic\WooCommerce\Admin\Install' ) ) {
-		require __DIR__ . '/src/Autoloader.php';
-
-		if ( \Automattic\WooCommerce\Autoloader::init() ) {
-			// Hook in WooCommerce Admin installation code.
-			// @todo an alternate way to do this is allow FeaturePlugin::has_satisfied_dependencies()
-			// to work during uninstall - (no WooCommerce instance).
-			\Automattic\WooCommerce\Admin\Install::drop_tables();
-		}
-	}
+if ( defined( 'WC_REMOVE_ALL_DATA' ) ) {
```

Comment:
I think it is better to restore the original behavior and check if the constant is defined and is set to true:

https://github.com/woocommerce/woocommerce/blame/9c8a6d78d9b358aeace95b39b10c9fcde363efcb/uninstall.php#L28

Reply:
Good idea, fixed in https://github.com/woocommerce/woocommerce/pull/25589/commits/42731d34e63b22f2f6384151859ca09c0316c07c

Reply:
LGTM, thanks!